decentraland/unity-explorer
# fix: added AudioSource to media player

This addresses this:
https://github.com/decentraland/unity-explorer/issues/758
And possibly this -> https://github.com/decentraland/unity-explorer/issues/999

## What does this PR change?

Our Media Players (for streaming music and video) dont use AudioSources so they dont get bundled up with our Audio Mixers and so we can't control their overall volume. This fixes that.
I just introduced a prefab that has all the required configuration and replaced the  generic MediaPlayer object we were spawning with this one.
This doesnt fix OTHER issues we had with MediaPlayers, but this opens the doors to fixing them :)

## How to test the changes?

Go to the ground floor of the Genesis Plaza and listen to the radio, try changing the WORLD volume on settings. The volume of the radio should be affected by these values.
